config/checker-framework-suppressions/checker-lock-tainting-suppressions.xml

```diff
@@ -447,6 +447,16 @@
     </details>
   </checkerFrameworkError>
 
+  <checkerFrameworkError unstable="false">
+    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ConstructorsDeclarationGroupingCheck.java</fileName>
+    <specifier>type.arguments.not.inferred</specifier>
+    <message>Could not infer type arguments for Optional.map</message>
+    <lineContent>.map(children::indexOf);</lineContent>
```

Comment:
Can we map to DetailAST.class::cast to avoid checker issue?

Reply:
@Zopsss , what is your response to this item ?

Reply:
Done:


https://github.com/checkstyle/checkstyle/blob/f1bf1f3c01c984ef487a06f56589ac75b9f66cbb/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ConstructorsDeclarationGroupingCheck.java#L107-L112

and sorry for the late reply, I was busy with college exam

Reply:
Ok, let’s just add checker suppression back and leave this battle for another day :)

Reply:
> leave this battle for another day :)

😂 Done. Added back the suppression